beluga/test/beluga/test_tuple_vector.cpp

```diff
@@ -177,6 +179,20 @@ TEST(TupleVectorTest, TraitConsistency) {
   static_assert(std::is_same_v<
                 ranges::range_rvalue_reference_t<Container>,  //
                 ranges::common_tuple<float&&, int&&>>);
+  static_assert(std::is_same_v<
+                ranges::range_value_t<ConstContainer>,  //
+                std::tuple<float, int>>);
+  static_assert(std::is_same_v<
+                ranges::range_reference_t<ConstContainer>,  //
+                ranges::common_tuple<const float&, const int&>>);
+  static_assert(std::is_same_v<
+                ranges::range_rvalue_reference_t<ConstContainer>,  //
+                ranges::common_tuple<const float&&, const int&&>>);
+
+  // Sadness... :(
```

Comment:
@nahueespinosa why so sad?

Reply:
I expected that the value type of the const view would be the same as the value type of the adapted container. But that's not the case. Added more context in https://github.com/Ekumen-OS/beluga/pull/288/commits/776e2b599b61775f267322fe909a571fc5932561.